src/Test/Utilities/AssertEx.cs

```diff
@@ -378,7 +378,7 @@ public static void Throws<T>(Action del, bool allowDerived = false)
                 Assert.Equal(typeof(T), type);
             }
 
-            Assert.True(false, "No exception was thrown.");
+            throw new Exception("No exception was thrown.");
```

Comment:
I wanted AssertEx.Throws to return the exception type a la xunit's regular Assert.Throws. The compiler's not smart enough to know that Assert.True will always throw, so this was the easiest way to shut it up.